app/src/main/java/com/codingblocks/cbonlineapp/mycourse/content/CourseContentFragment.kt

```diff
@@ -330,12 +332,18 @@ class CourseContentFragment : BaseCBFragment(), AnkoLogger, DownloadStarter {
                         LECTURE ->
                             if (contentLecture.lectureUid.isNotEmpty())
                                 startActivity(
-                                    intentFor<VideoPlayerActivity>(
-                                        CONTENT_ID to ccid,
-                                        SECTION_ID to sectionId
-                                    )
-                                )
-                            else
+                                    if (getPrefs()?.SP_PIP!!) {
```

Comment:
can you use prefs from ViewModel instead just to make the consistency throughout the code

Reply:
Sir everywhere it is used like getPrefs(requireContext()). So I used like this only